bin/repoxplorer-github-organization

```diff
@@ -97,8 +100,21 @@ if __name__ == "__main__":
             if r.name not in top:
                 continue
         data = {r.name: {"template": args.org}}
-        if r.default_branch != 'master':
```

Comment:
Why not only:
```
branches = set(r.default_branch)
if args.all_branches:
  for branch in r.branches():
    branches.add(branch)
branches = list(branch)

Reply:
Don't you mean?

`branches = list(branches)`

Reply:
Have simplified the code in line with your suggestion.

I'm still seeing issues though.